pyzabbix/sender.py

```diff
@@ -244,7 +240,13 @@ def _load_from_config(self, config_file):
         config_file_fp = StringIO(config_file_data)
         config = configparser.RawConfigParser(**params)
         config.readfp(config_file_fp)
-        zabbix_serveractives = config.get('root', 'ServerActive')
+        if config.has_option('root', 'ServerActive'):
```

Comment:
Please add comment here that we will use server if server is not exist but serveractive is. If both not exist default if ...  value will be used.